doc/interfaces/registration_flow.rst

```diff
@@ -0,0 +1,163 @@
+.. _registration_flow:
+
+============
+Registration
+============
+
+This tutorial walks through the steps to perform image-based and
+streamline-based registration using DIPY. Multiple registration methods are
+available in DIPY.
+
+You can try these methods using your own data; we will be using the data in
+DIPY. You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+------------------
+Image Registration
+------------------
+
+DIPY's image registration workflow can be used to register a moving image to a
+static image by applying different transformations, such as center of mass,
+translation, and rigid body or full affine (including translation, rotation,
+scaling and shearing) transformations. During such a registration process, the
+static image is considered to be the reference, and the moving image is
+transformed to the space of the static image. Registration methods use some
+sort of optimization process, and a given metric or criterion (like maximizing
+the mutual information between the two input images) that is optimized during
+the process, to achieve the goal.
+
+The DIPY image registration workflow applies the specified type of
+transformation to the input images, and hence, users are expected to choose the
+type of transformation that best matches the requirements of their problem.
+Alternatively, the workflows allows to perform registration in a progressive
+manner. For example, using affine registration with ``progressive`` set to
+``True`` will involve center of mass, translation, rigid body and full affine
+registration. Whereas, when ``progressive`` is ``False`` the registration will
+include only center of mass and affine registration. The progressive
+registration will be slower but will improve the quality.
+
+We will first create a directory in which to save the transformed image and the
+affine matrix (e.g.: ``image_reg_output``)::
+
+    mkdir image_reg_output
+
+To run the image registration, we need to specify the paths to the static image
+file, and to the moving image file, followed by the optional arguments. In this
+case, we will be specifying the type of registration to be performed
+(``transform``) and the output directory (``out_dir``).
+
+To perform center of mass registration, we will call the ``dipy_align_affine``
+command with ``transform`` parameter set to ``com`` e.g.::
```

Comment:
(...) with the {double_backticks}transform{double_backticks} parameter (...)

Reply:
Addressed in 997ec48